### PR TITLE
generate a few package tests

### PR DESCRIPTION
**Pull request by joshrwolf**
all of these were mostly (99%) generated with `gptscript` and the following:

https://gist.github.com/joshrwolf/a197145cfdccb73d2bbe4a780484a9a2

### CODE REVIEW

File: go-1.22.yaml [hunk 1]
```diff
@@ -92,3 +92,19 @@ update:
     strip-prefix: go
     tag-filter: go1.22
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+        - go-1.22
```
Comment (imjasonh) [inline go-1.22.yaml:101]: We don't need this explicitly

File: postgresql-16.yaml [hunk 2]
```diff
@@ -143,3 +143,33 @@ update:
     strip-prefix: REL_
     tag-filter: REL_16
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - wolfi-base
+        - postgresql-16
```
Comment (imjasonh) [inline postgresql-16.yaml:152]: Same

**Review by imjasonh — DISMISSED** [review]
I think we need some lint checks, but the tests lgtm

File: go-1.22.yaml (hunk 1 above)
Comment (joshrwolf) [reply go-1.22.yaml:101]: remnants courtesy of 🤖

File: go-1.22.yaml [hunk 3]
```diff
@@ -92,3 +92,18 @@ update:
...
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+  pipeline:
+    - name: Verify Go version
+      runs: |
+        go version | grep "go1.22"
```
Comment (ajayk) [inline go-1.22.yaml:104]:
```suggestion
        go version | grep "go${{package.version}}"
```

Comment (ajayk) [reply go-1.22.yaml:104]: Most of these are on auto version streams these will cause breakages in the test when we move to 1.23 automatically

**Pull request closed without merging**